src/api/app/assets/javascripts/webui2/staging_workflow.js

```diff
@@ -7,3 +7,19 @@ function setSpinnersForDeletion() { // jshint ignore:line
     $(this).find('.delete-spinner').addClass('d-none');
   });
 }
+
+$( document ).ready(function() {
+  $('.toggle-more').click(function(event) {
```

Comment:
what is it with you people's affinity to throw everything into one big function called everywhere?

Reply:
All of this can be done with [bootstrap collapse](https://getbootstrap.com/docs/4.1/components/collapse)

Reply:
See [this commit](https://github.com/openSUSE/open-build-service/pull/6265/files#diff-c9f6aa10ffbccedb7e93f3cac6e86ad3) for a way to do it with Bootstrap collapse. It's only about toggling classes. Everything else is handled by Bootstrap.

Reply:
@dmarcoux maybe you want to check this PR again. We are using collapse now, without any javascript.  I think we should sync tomorrow.

Reply:
@hennevogel 

> All of this can be done with bootstrap collapse

some custom css is needed, but yes.

Reply:
I use an icon for the link controlling the collapsible elements. However, I think for this PR, it's better how you did it.